mix.exs

```diff
@@ -19,7 +19,7 @@ defmodule Accounting.Mixfile do
       elixir: "~> 1.5",
       elixirc_paths: elixirc_paths(Mix.env),
       package: package(),
-      version: "0.10.5",
+      version: "0.11.0",
```

Comment:
Not a big deal since we're still pre 1.0.0, but I don't think that these changes broke any backwards compatibility.